### PR TITLE
Localize variables used in the editor in `common` instead of TEC

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Each line in an array declaration must end in a comma | [Assets.php:54](https://github.com/moderntribe/tribe-common/blob/aa272c597554ff55a107a42f85e4be4afee59ab8/src/Tribe/Editor/Assets.php#L54)

> via jenkins/codesniffer-phpcs

**Comment by mitogh** [comment]
Changes has been applied on this PR @bordoni

File: src/Tribe/Editor/Assets.php [hunk 1]
```diff
@@ -39,10 +41,23 @@ public function register() {
...
+					array(
+						'name' => 'tribe_editor_config',
+						/**
+						 * Array used to setup the FE with custom variables from the BE
+						 *
+						 * @since TBD
+						 *
+						 * @param array An array with the variables to be localized
+						 */
+						'data' => apply_filters( 'tribe_editor_config', $editor_configuration->localize() ),
```
Comment (bordoni) [inline src/Tribe/Editor/Assets.php]: Lets apply this filter on the Methods to pull it

**Comment by mitogh** [comment]
Changes has been applied thanks @bordoni

**Pull request merged**